Cargo.toml

```diff
@@ -12,19 +12,18 @@ repository = "https://github.com/maidsafe/BLS-DKG"
 version = "0.1.0"
 
 [dependencies]
-aes = "^0.3.2"
-bincode = "^1.1.4"
-block-modes = "^0.3.3"
-bytes = "0.4.12"
-crossbeam-channel = "0.3.9"
+aes = "~0.3.2"
+bincode = "1.2.1"
+block-modes = "~0.3.3"
+bytes = { version = "~0.5.4", features = ["serde"] }
+crossbeam-channel = "~0.4.2"
 ed25519_dalek = { package = "tmp-ed25519", version = "1.0.0-pre.3", features = ["serde"] }
-err-derive = "^0.2.4"
-itertools = "0.9.0"
+err-derive = "~0.2.4"
+itertools = "~0.9.0"
 log = "~0.4.8"
-rand = "^0.7.3"
-rand_core = "^0.2.1"
-schedule_recv = "0.1.0"
-serde = "^1.0.106"
-serde_derive = "^1.0.106"
-threshold_crypto = "^0.3.2"
-quic-p2p = "~0.5.0"
+rand = "~0.7.3"
+rand_core = "~0.2.1"
+serde = "~1.0.106"
+serde_derive = "~1.0.106"
```

Comment:
To bring into line with all other repo dependencies, all deps < 1.0.0 should contain a `~`, i.e. `~0.x.x`, while all deps >= 1.0.0 should be left with no decoration, i.e. `1.x.x`.

Therefore lines 26 & 27 should have their `~` removed

Reply:
ok, will change it